packages/javascript/js-client-library/src/client.ts

```diff
@@ -114,6 +117,52 @@ class BHEAPIClient {
     getAuditLogs = (options?: types.RequestOptions) => this.baseClient.get('/api/v2/audit', options);
 
     /* asset groups */
+
```

Comment:
I really like all the types here!! This will be so helpful for the rest of us outside of the initiative!